examples/amp-story/interactive_slider.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <meta name="description" content="This is the AMP Boilerplate." />
+    <link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link
+      rel="preload"
+      as="script"
+      href="https://cdn.ampproject.org/v0/amp-story-1.0.js"
+    />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+      async
+      custom-element="amp-story"
+      src="https://cdn.ampproject.org/v0/amp-story-1.0.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-story-interactive"
+      src="https://cdn.ampproject.org/v0/amp-story-interactive-0.1.js"
+    ></script>
+    <!-- Import other AMP Extensions here -->
+    <style amp-custom>
+      /* Add your styles here */
+      .title {
+        background-color: white;
+        border-radius: 60px;
+        padding: 40px;
+      }
+      h1 {
+        color: black;
+        font-size: 30px;
+        margin-bottom: 10px;
+        text-align: center;
+        font-family: 'Poppins';
+      }
+      p {
+        font-family: 'Poppins';
+        font-size: 15px;
+      }
+      .bottom {
+        position: absolute;
+        top: 80%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        width: 22em;
+      }
```

Comment:
Please remove style overrides on the component. 
We want to see the component without any style overrides on it.
This helps us see how the component will be styled out-of-the-box, with no configuration.

if you want to center the component on the page we can do that in a follow up PR.